packages/website/releases/1.4.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Release 1.4
+slug: 1.4
+authors:
+  - name: Rebecca Bol
+    title: Software Engineer | Data Visualization
+    url: https://github.com/reb-dev
+    image_url: https://avatars.githubusercontent.com/u/52078477
+image: https://unovis.dev/img/unovis-social.png
+hide_table_of_contents: false
+date: 2024-04-02T10:00
+---
+
+Version `1.4.0` of _Unovis_ is finally here! This update is packed full with enhancements including
+the new Annotations component, expanded Graph features, and a number of bug fixes to improve overall
+stability of the library.
+
+## Release Highlights
+### 📝 New component: Annotations
+Introducing _Annotations_, a versatile new component that enables you to overlay customized, stylized text on top of your visualizations. Whether you want to highlight points of interest, annotate trends, or simply add text labels to your charts or graphs, _Annotations_ is designed to integrate smoothly with any Unovis component. 
+
+Check out _Annotations_'s [documentation](https://unovis.dev/docs/auxiliary/Annotations) and [gallery example](https://unovis.dev/gallery/view?collection=Auxiliary%20Components&title=Chart%20Annotations) to learn how to use it.
```

Comment:
We can use absolute links without domain to make them work locally too, i.e. `/docs/auxiliary/Annotations`. 

Also if I remember it right, with absolute links Docusaurus can check if the linked documents really exist, which is nice when you re-organize the files.

P.S. I've also noticed that I put a wrong URL to `releases/1.3.md`. Can you please fix it too?
<img width="970" alt="image" src="https://github.com/f5/unovis/assets/755708/0a31cbd9-3d71-4fbc-b70e-33a21657b6db">